src/components/PokemonCard/PokemonCard.jsx

```diff
@@ -1,19 +1,38 @@
 import React from 'react';
 import { useHistory } from 'react-router';
+import style from './PokemonCard.css';
 
 export default function PokemonCard({ pokemon }) {
   const history = useHistory();
 
-  const { name, weight, types, id } = pokemon;
+  const { name, weight, types, id, sprites } = pokemon;
+
+  const arrayOfSprites = [];
+  const myFirstRecusion = (sprites) => {
```

Comment:
I'd put this in a utils folder

Reply:
ok, I'm actually having a hard time refactoring this one.